src/middleware/index.js

```diff
@@ -46,6 +46,7 @@ const readyStatePromise = store => next => action => {
 
 if (process.env.MIXPANEL_TOKEN) {
     mixpanel.init(process.env.MIXPANEL_TOKEN, {'property_blacklist': ['$current_url']})
+    window.mixpanel = mixpanel
```

Comment:
is this debug leftover?

Reply:
actually, not. I am thinking leaving this. But I will delete this for now since this is not for this PR.

Reply:
@icerove why is it necessary to keep `window.mixpanel`?

Reply:
I used to think of easy checking for us.